samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -247,7 +247,7 @@ select_the_feedback=Select the Feedback Components to be displayed to the studen
 selected=Selected
 selection_level_feed=Selection-level Feedback
 time_expired1=Time Expired!
-time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted.
+time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted in the next few minutes.
```

Comment:
Okay cool, I think your suggestion of "momentarily" would be better too. Something like:

"The time allowed for this assessment has expired. Your answers will be automatically submitted momentarily."